lib/ansible/modules/cloud/vmware/vmware_cluster_drs.py

```diff
@@ -70,6 +70,12 @@
       type: int
       default: 3
       choices: [ 1, 2, 3, 4, 5 ]
+    advanced_settings:
+      version_added: "2.10"
+      description:
+      - A dictionary of advanced HA settings.
```

Comment:
The definition is rather vague and I don't want to see horde of confused users on #ansible-vmware :-). Could you please either add an example or extend the description.

Reply:
Also, these settings are related to DRS.

Reply:
I've added an example and fixed the C&P error `HA` to `DRS`.